src/gui/editors/BBEditor.cpp

```diff
@@ -311,3 +314,21 @@ void BBTrackContainerView::makeSteps( bool clone )
 		}
 	}
 }
+
+// Creates a clone of the current BB track with the same pattern, but with no TCOs on the song editor
```

Comment:
```suggestion
// Creates a clone of the current BB track with the same pattern, but no TCOs in the song editor
// TODO: Avoid repeated code from cloneTrack and clearTrack in TrackOperationsWidget somehow
```
1. Removed second `with`. This is super nitpicky, but I think it's shorter and more natural.
2. `no TCOs on the song editor` -> `no TCOs in the song editor`. You could even drop `in the song editor` entirely.
3. I think it'd be nice to leave a TODO here for future reference.